src/libostree/ostree-core.h

```diff
@@ -158,6 +158,16 @@ typedef enum {
  * - a(s(taya{sv})) - Map of ref name -> (latest commit size, latest commit checksum, additional metadata), sorted by ref name
  * - a{sv} - Additional metadata, at the current time the following are defined:
  *   - key: "ostree.static-deltas", value: a{sv}, static delta name -> 32 bytes of checksum
+ *   - key: "ostree.summary.last-modified", value: t, timestamp (seconds since
+ *     the Unix epoch in UTC) when the summary was last regenerated (similar to
+ *     the HTTP `Last-Modified` header)
```

Comment:
And we need this for cases where we're not fetching over HTTP?  At least in the HTTP case we could get the value from the headers and store it in an xattr or so.

But, I guess if we need to support non-HTTP, it's OK.